rules/falco_rules.yaml

```diff
@@ -2764,7 +2769,12 @@
     "stratum-ltc.antpool.com",
     "stratum-zec.antpool.com",
     "stratum.antpool.com",
-    "xmr.crypto-pool.fr"
+    "xmr.crypto-pool.fr",
+    "ss.antpool.com",
+    "stratum-dash.antpool.com.antpool.com",
```

Comment:
should `stratum-dash.antpool.com.antpool.com` be `stratum-dash.antpool.com`

Reply:
Thank you. Typo fixed.